satochip-lib/src/main/java/org/satochip/client/Bip32PathToBytesObject.java

```diff
@@ -0,0 +1,20 @@
+package org.satochip.client;
+
+public class Bip32PathToBytesObject {
```

Comment:
Rename this class to Bip32Path and store depth, bytes and string format of a Bip32Path, with function to convert from string to byte[] (as implemented in SatochipParser.parseBip32PathToBytes(String bip32path) )

Add a constructor that can take the stringPath as argument.